_posts/2020-11-19-Pull-Request-Set-up.md

```diff
@@ -6,13 +6,7 @@ summary:
 tags:
 ---
 Kajari instructed Osvaldo and I to start creating a new post using pull request. Pull requests are changes committed in a branch that are presented to viewers of your repository in github. To initiate a new post/pull request, the following commands are necessary for committing the task: 
```

Comment:
You don't need to talk about "Kajari instructing us" to do anything. Remove this line and start on the next line talking about pull requests. Or if you want an introductory line, you can start with "today we started learning about pull requests."